product/reports/200_Red Hat/001_CPU Right Sizing Recommendations.yaml

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
I think the name of the file should not contain spaces, but underscores instead. Also I'm not really suure about the `Red Hat` in the folder's name.

Reply:
Okay, so the naming format is the following:
```
###_Category with spaces - Subcategory/###_Name of the report with spaces.yaml
```
Otherwise your report will not be displayed.

Reply:
I fixed the name for the folder. Red Hat name was a suggestion from Loic to mimic what we do in Automation, where we have ManageIQ and Red Hat datastores.

Thanks for the naming tip!